controller/src/main/java/io/pravega/controller/store/stream/CreateStreamResponse.java

```diff
@@ -7,13 +7,21 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
-package io.pravega.controller.store.stream.tables;
+package io.pravega.controller.store.stream;
 
 import io.pravega.client.stream.StreamConfiguration;
 import lombok.Data;
 
 @Data
-public class Create {
-    private final long creationTime;
+public class CreateStreamResponse {
+    public enum CreateStatus {
+        NEW,
```

Comment:
this is wrt creation of state in metadata. 
New captures if the stream created is a new one or already exists.